src/app/youtube/components/search-item/search-item.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-search-item',
+  templateUrl: './search-item.component.html',
+  styleUrls: ['./search-item.component.css']
+})
+export class SearchItemComponent {
+
+
```

Comment:
remove all this `@Input()`
and add only one @Input() currentSearchItem;

Reply:
I removed